packages/neon_framework/lib/src/theme/icons.dart

```diff
@@ -0,0 +1,130 @@
+// ignore_for_file: non_constant_identifier_names
+
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_material_design_icons/flutter_material_design_icons.dart';
+import 'package:neon_framework/src/utils/adaptive.dart';
+
+/// A set of platform-adaptive icons.
+abstract final class AdaptiveIcons {
+  const AdaptiveIcons._();
+
+  /// Platform-adaptive icon for
+  /// <i class="material-icons md-36">search</i> &#x2014; material icon named "search"
+  /// and <i class='cupertino-icons md-36'>search</i> &#x2014; cupertino icon named "settings".;
```

Comment:
So this is special html to make the icons appear in the docs or how does that work?

Reply:
copy paste from the upstream docs (the CupertinoIcons and Icons one).
I also have no idea how it works :P

Reply:
Haha ok, I hope it is not too hard to find for new icons, maybe you can explain in a comment at the top of the file how to get this html.

Reply:
it's fairly simple `<i class="material-icons md-36">$iconName</i>`
I'll add a comment nevertheless.